test/js/computed-style-property-map.js

```diff
@@ -23,6 +23,18 @@ suite('Computed StylePropertyMap', function() {
     assert.strictEqual(propertyStyleValue.cssString, '0.5');
   });
 
+  test('get method returns the first StyleValue in the sequence if a property has been set a sequence ' +
+    'of StyleValues', function() {
+    var inlineStyleMap = this.element.styleMap();
+    var computedStyleMap = getComputedStyleMap(this.element);
+    var valueArray = [new NumberValue(4), new NumberValue(5), new KeywordValue('infinite')];
+    inlineStyleMap.set('animation-iteration-count', valueArray);
+    var propertyStyleValue = computedStyleMap.get('animation-iteration-count');
+
+    assert.instanceOf(propertyStyleValue, NumberValue);
+    assert.strictEqual(propertyStyleValue.cssString, '4');
+  });
```

Comment:
It would also be great to have a test for the parse method directly - you can create a new test file for StyleValue.